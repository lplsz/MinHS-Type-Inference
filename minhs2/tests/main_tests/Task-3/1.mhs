main
     = let x :: Bool = True;
           y :: Int = 1;
           z = ();
       in x;
