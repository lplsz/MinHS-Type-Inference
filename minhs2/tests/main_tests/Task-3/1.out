main :: Bool = let x :: Bool = True;
                   y :: Int = 1;
                   z :: () = ();
               in x;